tox.ini

```diff
@@ -1,27 +1,21 @@
 [tox]
 envlist =
+    django111-{py27,py3}
     django110-{py27,py3}
     django19-{py27,py3},
-    django18-{py27,py3},
-    django17-{py27},
-    django16-{py27},
-    django15-{py27},
-    django14-{py27},
 
-[testenv:django15-py27]
-commands = {envpython} manage.py test object_tools --settings=test_settings
+[testenv:django19-py27]
+commands =
+	{envpython} manage.py test --settings=object_tools.tests.settings.19
 
-[testenv:django14-py27]
-commands = {envpython} manage.py test object_tools --settings=test_settings
+[testenv:django19-py3]
```

Comment:
`testenv` is the default that runs for 110 and 111.
Although, I'll add separate settings files for 110 and 111 and I'll then have to name the `testenv`s explicitly